### ISSUE TITLE
Where are the logs?

### ISSUE THREAD

**Issue by gustiando** [hotwired/turbo-rails#332]
Hi, I've been failing to get a relatively simpler version of Turbo Stream working in my app and I'm wondering why.

I can go in more detail of what I got setup, but ultimately, if I supposedly got the whole thing wired up. How can I understand why something isn't working?

Nothing errors out and nothing happens other than seeing my turbo stream response coming from the server and my target sitting there in the html without it getting "matched" and replaced with.

Are the some verbose logs that I can enable? or how should I go about troubleshooting it?

Thanks!

**Comment by noahc** [comment]
What do you see in the rails server logs? What do you see the console logs of your browser? Do they match your expectations? Why or why not?

**Comment by tleish** [comment]
Can you share your project (e.g. on a github repo)?

**Comment by gustiando** [comment]
@tleish unfortunately I cannot because it's a private repo. Do you have any pointers on what piece of the puzzle I'm missing?

I might give another go soon again, but things seem to be still where I left off. Was hoping to share some good news with the team that are looking forward to get their hands on Hotwire to see how it "feels". We don't seem to have tons of experience with it so any presumably simple working example could help us out, which is why :)

**Comment by tleish** [comment]
If you can create a separate demo repo where you reproduce the issue, then I might be able to help.

**Comment by marcoroth** [comment]
Hey @gustiando, just checking back, were you able to resolve this issue in the meantime? 

If not, I would be more than happy to guide you through a possible solution. Thank you!

**Comment by gustiando** [comment]
@marcoroth unfortunately not. I haven't gotten a chance to create a separate demo repo and all that yet. Got distracted and now will need to wait for another opportunity to touch turbo again at work. So I'll close this for now and revisit if/when i get a chance. Thanks for reaching out!

**Comment by dacook** [comment]
> Are the some verbose logs that I can enable? or how should I go about troubleshooting it?

This is the issue, and as far as I'm concerned, it's unresolved.

Somebody created a gem to highlight Turbo elements, and log each event, which I've found very helpful: https://github.com/beflagrant/hotwire-debug
I suggest this could be incorporated into the main gem, hidden behind a debug config option.

In particular, I would find it helpful if any element with `data-turbo*` attributes was highlighted. I had an issue where Turbo was disabled by an ancestor element, and I wasted a lot of time before realising this!
